quemsoueu.html

```diff
@@ -0,0 +1,84 @@
+
+     <!--- Primeira linha ------------------->
+     <div class="banner__superior">
+        <div class="logo__superior">
+            <img src="./img/logovinho.jpeg" class="logo__imagem">
+        </div>
+        <nav class="menu__superior">
+            <ul class="menu__superior__grupo">
+                <li><a href="paineis.html" class="menu__superior--itens">Painéis</a></li>
+                <li><a href="suporte.html" class="menu__superior--itens">Suporte de Plantas</a></li>
+                <li><a href="espelho.html" class="menu__superior--itens">Espelhos</a></li>
+                <li><a href="balanços.html" class="menu__superior--itens">Balanços</a></li>
+                <li><a href="bolsas.html" class="menu__superior--itens">Bolsas</a></li>
+                <li><a href="arvore da vida.html" class="menu__superior--itens">Árvores da vida</a></li>
+                <li><a href="quemsoueu.html" class="menu__superior--itens">Quem sou eu</a></li>
+            </ul>
+        </nav>
+
+    </div>
+
+       <!--- Quadrado Propaganda ------------------->
+       <div class="quadrado__propaganda">
+            <img src="./imgdosprodutos/testebanner1.png" class="imagem--banner" alt="Imagem 1">
+            <img src="./imgdosprodutos/testebanner2.png" class="imagem--banner" alt="Imagem 2">
+            <img src="./imgdosprodutos/testebanner3.png" class="imagem--banner" alt="Imagem 3">
+        </div>
+
+    <div class="divisoria"></div>
+
+    <div class="novidades">Quem sou eu</div>
+
+    <div class="novidades__produto">
+        <div class="novidades__produto--01">
+            <div class="novidades__produto--01--info">
+                <p class="novidades__produto--01--txt quemsoueu__titulo">Ilse Barradas</p>
+                <p class="novidades__produto--01--txt quemsoueu__txt">bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla bla</p>
+            </div>
+        </div>
+        <div class="novidades__produto--02">
+            <img src="./imgdosprodutos/Arvore da Vida/arvoredavida2.JPG" class="novidades__produto--01--imagem">
+        </div>
+    </div>
+
+     <!--- Footer ------------------->
+
+     <footer>
+        <div class="divisoria"></div>
+
+        <div class="footer__info">
+            <div class="footer__image">
+                <img src="./img/logovinho.jpeg" class="footer__image--01">        
+            </div>
+            <div class="footer__info--contato">
+                <p class="footer__info--contato--txt">CONTATO</p>
+                <a href="https://instagram.com/ilsebarradas1?igshid=Y2IzZGU1MTFhOQ==" class="footer__info--contato--imagem">
+                    <img src="./img/instagram-logo.png" alt="instagram" class="footer__info--contato--imagem">
+                </a>
+            </div>
+        </div>
+
+        
+
+        <div class="footer__dev">
+            <p class="footer__dev--text">Desenvolvido por Augusto Barradas 2023</p>
```

Comment:
Já coloca um tag `a` aqui pra vc adicionar o link pra o seu portifólio depois
```suggestion
            <p class="footer__dev--text">Desenvolvido por <a hreft="">Augusto Barradas</a> 2023</p>
```

Reply:
Ok